README.md

```diff
@@ -43,11 +42,24 @@ See the command line options with:
 
 ### SSL/TLS
 
-`feed-ingress` expects your ELBs to terminate SSL traffic. We believe this is the safest and best performing
+## SSL termination on ELB
+SSL termination could be done on ELBs, probably the safest and best performing
 approach for production usage. Unfortunately, ELBs don't support SNI at this time, so this limits SSL usage to
 a single domain. One workaround is to use a wildcard certificate for the entire zone that `feed-dns` manages.
 Another is to place an SSL termination EC2 instance in front of the ELBs.
 
+## SSL termination on feed-ingress
+
+SSL termination could be done on feed-ingress. You will be able handle the secure websocket and HTTPS with a layer 4 in front (ELB, IPVS(GORB)).
```

Comment:
"can be done" ... "This approach still requires a layer 4 load balancer, eg. ELB or IPVS with GORB, in front".